randovania/games/dread/json_data/Elun.txt

```diff
@@ -519,7 +519,7 @@ Extra - asset_id: collision_camera_007
   > Door to Chozo Soldier Arena
       All of the following:
           Can Slide
-          Grapple Beam or Spider Magnet or Space Jump or Speed Booster or Walljump (Beginner) or Simple IBJ
+          Grapple Beam or Spider Magnet or Space Jump or Speed Booster or Simple IBJ
```

Comment:
this whole thing is trivial except for Can Slide. Just needs to be a Template for it